frontend/src/routes/AdminPage/index.js

```diff
@@ -30,30 +30,27 @@ import {
   GroupLogoWrapper,
 } from "./styles";
 
-class AdminPage extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      results: undefined,
-      error: null,
-      user: { name: "" },
-      applications: [],
-      csvData: [],
-      isFetching: true,
-      headers: [
-        { label: "Full Name", key: "name" },
-        { label: "Søknadstekst", key: "applicationText" },
-        { label: "Mobilnummer", key: "phoneNumber" },
-        { label: "Email", key: "email" },
-        { label: "Username", key: "username" },
-        { label: "Søkt innen frist", key: "appliedWithinDeadline" },
-        { label: "Tid sendt", key: "createdAt" },
-        { label: "Tid oppdatert", key: "updatedAt" },
-      ],
-    };
-  }
+const AdminPage = ({ groups }) => {
+  const [state, setState] = useState({
```

Comment:
I guess this works fine, but it doesn't follow convention on how we usually use `useState`. Is there any reason not to split it up into separate `useState`s?

Reply:
The reason I initially didn't do it is that the next step in my process is to implement `react-query` for data-fetching and caching, and that should enable the removal of many of the bigger state objects. So in that way it felt more practical to not change too much in this PR and take care of that in the next.

Reply:
I see, that makes sense

Reply:
But still, it doesn't take that long to fix it, and I appreciate the thorough review so I updated it now:)